platform-api/src/routers/discord.py

```
@@ -0,0 +1,74 @@
+        "title": body.title,
+        "description": body.message,
+        "color": COLOR_PRESETS.get(body.color, COLOR_PRESETS["blue"]),
+        "timestamp": datetime.now(timezone.utc).isoformat(),
+    }
+    if body.fields:
+        embed["fields"] = [f.model_dump() for f in body.fields]
+
+    # 4. Send (Sync or Async)
+    if body.async_mode:
+        # Fire-and-forget: Return 202 immediately, send in background
+        asyncio.create_task(post_webhook(webhook_url, embed))
```

Ruff (0.15.11)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)